src/client/unittests/main.ts

```diff
@@ -42,17 +43,14 @@ export class UnitTestManagementService implements IUnitTestManagementService, Di
             this.workspaceTestManagerService.dispose();
         }
     }
-    public async activate(): Promise<void> {
+    public async activate(symboldProvider: vscode.DocumentSymbolProvider): Promise<void> {
```

Comment:
You choose whether or not to fix this one
```suggestion
    public async activate(symbolProvider: vscode.DocumentSymbolProvider): Promise<void> {
```